test_argparse.c

```diff
@@ -25,6 +26,7 @@ main(int argc, const char **argv)
         OPT_BOOLEAN('t', "test", &test, "test only"),
         OPT_STRING('p', "path", &path, "path to read"),
         OPT_INTEGER('n', "num", &num, "selected num"),
```

Comment:
Shouldn't this be re-named to `"int"` or sth. now? The choice `"num"`/`"float"` seems a bit confusing now that there are several numeric options.

Reply:
You're right. I'll change that and I'll also update the unit tests. I'll submit another pull request with these changes.

Reply:
Done. It seems that I don't need to create another pull request. My latest commit appears in this one.